server/routes/journeys/dailySchedule/journey.ts

```diff
@@ -6,4 +6,5 @@ export type ScheduleFilters = {
   period?: Period[]
   appointmentLocation?: string[]
   courtOrProbationTeam?: string[]
+  caseLoadId?: string | undefined
```

Comment:
Need this to be stored on the filter so we can clear the filter session should the case load change.  This was handled previously in removed middleware.